src/shared/components/cancerSummary/SummaryBarGraph.tsx

```diff
@@ -115,6 +119,13 @@ export default class SummaryBarGraph extends React.Component<ISummaryBarGraphPro
     }
 
     public componentDidMount() {
+        (Chart as any).plugins.register({
```

Comment:
seems like this is applied to all charts then?  
if so, we should see if there's a different way to accomplish this.